Tests/scripts/collect_tests/utils.py

```diff
@@ -126,6 +125,23 @@ def _calculate_to_version(self) -> Version | InfinityType:
             return Version(value)
         return version.Infinity
 
+    def _handle_xsoar_marketplaces(self) -> tuple[MarketplaceVersions, ...] | None:
```

Comment:
can we import the SDK method that does it, instead of reimplementing here?

Reply:
We have this code in the SDK but as part of an init function and not as a separate method that can be imported at this moment. I will modify the SDK code to take this functionality outside to an importable function.
Will modify here accordingly as a phase 2 and separate PR.

Reply:
ok, please open a jira for that